### ISSUE TITLE
User Story 1 & 2 notepad

### ISSUE THREAD

**Issue by LibbyUX** [hubmapconsortium/hra-ui#576]
@andreasbueckle, I am starting an idea notepad for us here regarding User Story 1 & 2 UIs. Please capture all feature requests, UI inspiration screenshots, links, and more in this area!

**Comment by andreasbueckle** [comment]
Pretty neat use of button to cite the tool (could link to HRApop DOI once available):
<img width="1337" alt="image" src="https://github.com/user-attachments/assets/60de0900-5724-493c-bec8-b323de3c45d3">

**Comment by LibbyUX** [comment]
@andreasbueckle:

Last night I contemplated branding for these user story preview/beta tools. Concrete tool names and custom icon designs (see: [tool icons](https://www.figma.com/design/gQEMLugLjweDvbsNNUVffD/Design-System-Repository?node-id=2033-1383&t=s3gbY51168zy7iLo-4)) could elevate them when the time comes to revamp the features of these UIs.

Screenshot list of our current user stories in preview/beta mode:
![image](https://github.com/user-attachments/assets/d84eace8-96c6-4113-b139-2faf7f5cd7a7)

**Comment by andreasbueckle** [comment]
@LibbyUX 
Yeah, that might be worth revisiting for US #1-2 especially. Also, a clearer link to HRA US numbers these previews are serving would be good.

**Comment by LibbyUX** [comment]
@andreasbueckle - We are getting closer to needing official names for these two products as the Atlas Applications design matures. Do you have any ideas off the top?

### Atlas Apps Menu Design
Note: This will change. We are adding screenshots to make it more visual instead of having the product icons.
![image](https://github.com/user-attachments/assets/430c1996-258f-415a-a5c9-c77af8b3f336)

**Comment by LibbyUX** [comment]
@andreasbueckle Can we add features to already existing products to consolidate our products? Or do you see these two user stories as requiring their own product identity?

### HRA Product Line
- Figma - [Information Architecture: Products](https://www.figma.com/board/H3CK8StFdAphCFtFAHAwhM/Information-Architecture?node-id=479-1486&t=OddD7ooQ6GXHbEGc-4)

![image](https://github.com/user-attachments/assets/65cdff77-96f3-41aa-9296-cfd478562503)

**Comment by andreasbueckle** [comment]
> @andreasbueckle - We are getting closer to needing official names for these two products as the Atlas Applications design matures. Do you have any ideas off the top?
> ### Atlas Apps Menu Design
> 
> Note: This will change. We are adding screenshots to make it more visual instead of having the product icons. ![image](https://private-user-images.githubusercontent.com/97060856/358596860-430c1996-258f-415a-a5c9-c77af8b3f336.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwOTc0MDQsIm5iZiI6MTcyNDA5NzEwNCwicGF0aCI6Ii85NzA2MDg1Ni8zNTg1OTY4NjAtNDMwYzE5OTYtMjU4Zi00MTVhLWE1YzktYzc3YWY4YjNmMzM2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDE5NTE0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQxNzQyZmVlNWU0MDJhOTlkNGNlYzJkZTMyYWI2NjY4YWE1ZDBiYjA0M2UyNmY0YWM0YjlkM2IzODhiMDIxMDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-dfLQIRivR4gHXRtY-I9wLxK6glQEUkL6jxTmpT8CMU)

I like the two suggestions you have on the right. @bherr2 could we rename https://apps.humanatlas.io/us1/ and https://apps.humanatlas.io/us2/ to match those names?

**Comment by andreasbueckle** [comment]
> @andreasbueckle Can we add features to already existing products to consolidate our products? Or do you see these two user stories as requiring their own product identity?
> ### HRA Product Line
> 
>     * Figma - [Information Architecture: Products](https://www.figma.com/board/H3CK8StFdAphCFtFAHAwhM/Information-Architecture?node-id=479-1486&t=OddD7ooQ6GXHbEGc-4)
> 
> 
> ![image](https://private-user-images.githubusercontent.com/97060856/359204312-65cdff77-96f3-41aa-9296-cfd478562503.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwOTc0MDQsIm5iZiI6MTcyNDA5NzEwNCwicGF0aCI6Ii85NzA2MDg1Ni8zNTkyMDQzMTItNjVjZGZmNzctOTZmMy00MWFhLTkyOTYtY2ZkNDc4NTYyNTAzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDE5NTE0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMzZDNjY2EyZTNhZTZmZTk0ZDM2N2U3ZTc3NjFmNTRmMGExOGZhOGUyMzY3YTlmMDkwNWFlOWUyY2ZlNzEyNzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4wa9Kj2BF25iyfto2DKEDrW8eCfKrEdztS6rNbhdtac)

I think these two user stories need their own identity. These are demo apps that we are documenting with the paper. There will probably be other ways to access the data, and the HRA API also allows the usage of HRApop data for prediction.

**Comment by LibbyUX** [comment]
Hey Andi, regarding:
> I like the two suggestions you have on the right. @bherr2 could we rename https://apps.humanatlas.io/us1/ and https://apps.humanatlas.io/us2/ to match those names?

If they are having their own product identity, they will need strong names, branding, etc. I created the [HRA Product Line board](https://www.figma.com/design/gQEMLugLjweDvbsNNUVffD/Design-System-Repository?node-id=5447-20851&t=idWtNMxZPcHsVEGP-4) in our Figma Design System file to bring order to our HRA internal/external product jargon, naming inconsistencies, and product identities. US1 & US2 can be found in the [Emerging Products](https://www.figma.com/design/gQEMLugLjweDvbsNNUVffD/Design-System-Repository?node-id=5452-21409&t=idWtNMxZPcHsVEGP-4) section. 

Can you please analyze the full board in Figma along with the US1 & US2 entries and let me know what you think?

![image](https://github.com/user-attachments/assets/95dc4903-1383-4a33-aeba-0f2c70af6343)
